### PR TITLE
Readme: Version Update Rules

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**
Set of rules for Unified Schema Versioning on the Notices producers document:
(a) Includes version type
(b) Announce to team and community
(c) Mission page rule

Resolves #https://github.com/nasa-gcn/gcn-schema/issues/142

### CODE REVIEW

File: gcn/notices/core/Alert.schema.json
Comment (lpsinger) [inline gcn/notices/core/Alert.schema.json:1]: Why did you remove this file?

File: README.md [hunk 1]
```diff
@@ -33,9 +33,9 @@ npm install
 
         npm install
 
-3.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules.
+3.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules. Updates are categorized as major, minor, or patch, depending on the type of change. A major change is defined as any modification that breaks compatibility with the core schema, such as a change in a property or the addition of a new property that affects the mission pipelines. Introducing a new mission and new core schema constitutes a minor change, while modifying the new mission schema and adding a new property to the core schema are considered patches.
```
Comment (lpsinger) [inline README.md:36]: This just looks like a summary of SemVer. What is the value of adding it?

Comment (Vidushi-GitHub) [reply README.md:36]: This includes examples and we prefer documents to be self-consistent for clarity. 

@jracusin Do you also think it's not adding any value?

Comment (lpsinger) [reply README.md:36]: Alright. Some corrections:

- Major revisions are for changes that break compatibility; they are not necessarily limited to core schema changes.
- Minor revisions are for introduction of new features: new schema, or new fields in existing schema.
- Patch revisions are for bug fixes that do not break compatibility.

Comment (dakota002) [reply README.md:36]: Can you also include a note clarifying that we will run these steps, and that producers should ignore it? I have seen a couple times now of new producers running the version command in their PRs

Comment (Vidushi-GitHub) [reply README.md:36]: Yes, thanks. I will add a note here and will add a line in Notices producers document as well. I have also seen this confusion.

File: gcn/notices/core/Alert.schema.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
     },
     "alert_type": {
       "enum": ["initial", "update", "retraction"],
-      "description": "Indication of alert sequence if multiple of same type sent"
+      "description": "Indication of alert sequence if multiple of the same type are sent."
```
Comment (lpsinger) [inline gcn/notices/core/Alert.schema.json:25]: Please submit this unrelated typographical correction in a separate PR.

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Comment by Vidushi-GitHub** [comment]
Re-arranged and added a note regarding tagging the version Screenshot 2024-12-05 at 11 59 09 PM

File: README.md (hunk 1 above)
Comment (Vidushi-GitHub) [reply README.md]: > Can you also include a note clarifying that we will run these steps, and that producers should ignore it? I have seen a couple times now of new producers running the version command in their PRs

Done, see below screenshot.

File: README.md [hunk 3]
```diff
@@ -33,13 +33,19 @@ npm install
 
         npm install
 
-3.  Tag a new version by running the following command: `npm version [ major | minor | patch ]`. Choose `major`, `minor`, or `patch` depending on the kind of update according to [Semantic Versioning](https://semver.org) rules.
+## Tag a Version
```
Comment (lpsinger) [inline README.md:36]: Please turn this back into an enumerated list item. When you made it into a heading, it interrupted the logical flow of the document.

File: README.md [hunk 4]
```diff
@@ -50,3 +56,5 @@ This will make the release available for use. For subsequent releases:
         git pull
 
 2.  Follow the above steps from Step 3 on.
+
+**Note**: Tagging the version will be handled by the maintainers and should be ignored by producers.
```
Comment (lpsinger) [inline README.md:60]: "Ignored" isn't the right word here. Tags are important for producers because producers should be using the tagged version of their schema.

**Review by lpsinger — CHANGES_REQUESTED** [review]

**Review by lpsinger — APPROVED** [review]
